README.md

```diff
@@ -1,17 +1,16 @@
-# Deno Docker
+# deno_docker
 
-Docker files for [deno](https://github.com/denoland/deno) published on Dockerhub:
+Docker files for [Deno](https://github.com/denoland/deno) published on
+Dockerhub:
 
-- Alpine Linux: [hayd/alpine-deno](https://hub.docker.com/r/hayd/alpine-deno/) (\~28Mb)
-- Centos: [hayd/centos-deno](https://hub.docker.com/r/hayd/centos-deno/) (\~94Mb)
-- Debian: [hayd/debian-deno](https://hub.docker.com/r/hayd/debian-deno/) (\~47Mb)
-- Distroless: [hayd/distroless-deno](https://hub.docker.com/r/hayd/distroless-deno) (\~28Mb)
-- Ubuntu: [hayd/ubuntu-deno](https://hub.docker.com/r/hayd/ubuntu-deno/) (\~50Mb)
+- Alpine Linux: [denoland/deno:alpine](https://hub.docker.com/r/denoland/deno)
+  (default)
+- Centos: [denoland/deno:centos](https://hub.docker.com/r/denoland/deno)
+- Debian: [denoland/deno:debian](https://hub.docker.com/r/denoland/deno)
+- Distroless: [denoland/deno:distroless](https://hub.docker.com/r/denoland/deno)
+- Ubuntu: [denoland/deno:ubuntu](https://hub.docker.com/r/denoland/deno)
```

Comment:
Historical note: I had these as ubuntu-deno etc. since that way the popular (but not recommended) :latest tag is stable in the os it uses.
Perhaps there is a better workaround!

Reply:
The way I set it up now is as follows:

All base OS images are available at these tags on each release:
- denoland/deno:${BASE}
- denoland/deno:${BASE}-${DENO_VERSION}

Debian is available on these tags:
- denoland/deno:latest
- denoland/deno:${DENO_VERSION}

That way the following tags would resolve as follows, assuming the latest release is 1.10.2:

- `denoland/deno` -> debian 1.10.2
- `denoland/deno:debian` -> debian 1.10.2
- `denoland/deno:debian-1.10.2` -> debian 1.10.2
- `denoland/deno:1.10.2` -> debian 1.10.2
- `denoland/deno:debian-1.10.1` -> debian 1.10.1
- `denoland/deno:1.10.1` -> debian 1.10.1
- `denoland/deno:ubuntu` -> ubuntu 1.10.2
- `denoland/deno:ubuntu-1.10.2` -> ubuntu 1.10.2
- `denoland/deno:ubuntu-1.10.1` -> ubuntu 1.10.1